meta-mel/conf/distro/include/mel.conf

```diff
@@ -355,63 +357,9 @@ IMAGE_CLASSES += "archive_sdk_downloads"
 # Create a 'latest' symlink for the SDK
 IMAGE_CLASSES += "sdk_latest_link"
 
-DATETIME_SECS = "${@int(time.mktime(time.strptime('${DATETIME}', '%Y%m%d%H%M%S')))}"
-DATETIME_SECS[doc] = "The timestamp from the DATETIME variable, as seconds since 1970."
-
-# Configure the ADE
-DEPLOY_DIR_ADE ?= "${DEPLOY_DIR}/ade"
-ADE_PROVIDER ?= "Mentor Graphics Corporation"
-# The ADE_VERSION has to stay numeric with . separators
-OE_IMPORTS += "string"
-ADE_VERSION ?= "${@'0.0' if not all(c in string.digits for c in '${SDK_VERSION}'.split('.')[0]) else '${SDK_VERSION}'}.${DATETIME_SECS}"
-ADE_VERSION[vardepsexclude] += "DATETIME_SECS"
-ADE_IDENTIFIER ?= "${SDKMACHINE}-${MLPREFIX}${IMAGE_BASENAME}-${MACHINE}-${ADE_VERSION}"
-ADE_IDENTIFIER_SUBDIR = "/ade-${ADE_IDENTIFIER}"
-ADE_SITENAME ?= "ADE for ${ADE_IDENTIFIER}"
-ADE_TOP_CATEGORY ?= "${DISTRO_NAME} ADE for ${IMAGE_BASENAME}-${MACHINE}"
-
-# Configure populate_sdk_p2 for the ADE
-SDK_P2_DEPLOY = "${DEPLOY_DIR_ADE}"
-SDK_P2_OUTPUTNAME = "ade-${ADE_IDENTIFIER}"
-SDK_P2_REPO = "${SDK_P2_DEPLOY}/${SDK_P2_OUTPUTNAME}"
-SDK_P2_INSTALL_PATH = "/../../mel-flex-os/ade${ADE_IDENTIFIER_SUBDIR}"
-SDK_P2_DOCKER_LOCATION = "/ade"
-SDK_P2_TAR_FILENAME = "ade"
-SDK_P2_TITLE = "${ADE_SITENAME}"
-SDK_P2_CATEGORY = "${ADE_TOP_CATEGORY}"
-SDK_P2_CATEGORY_NAME = "${ADE_TOP_CATEGORY}"
-SDK_P2_VERSION = "${ADE_VERSION}"
-SDK_P2_TIMESTAMP = "${DATETIME_SECS}"
-SDK_P2_TIMESTAMP[vardepsexclude] += "DATETIME_SECS"
-SDK_P2_ZIP_EXTRA_FILES = "Dockerfile cb-mbs-options*"
-
-# Fixup these vars to use correct values for the image
-SDK_RESET_ENV_SCRIPT_VARS += "\
-    ADE_IDENTIFIER \
-    ADE_SITENAME \
-"
-
```

Comment:
where these vars has moved now?

Reply:
https://github.com/MentorEmbedded/mel-meta-mentor-private/pull/134

Reply:
thanks i missed that particular commit at the end.